modules/Core/pages/profile.php

```diff
@@ -108,6 +108,18 @@
                                 ]
                             );
 
+                            EventHandler::executeEvent('userNewProfilePost', [
+                                'username' => $user->getDisplayname(true),
+                                'content' => $language->get("user", "x_posted_on_y_profile", [
+                                    "poster" => $user->getDisplayname(),
+                                    "user" => $query->username
+                                ]),
+                                'content_full' => strip_tags(str_ireplace(['<br />', '<br>', '<br/>'], "\r\n", Input::get('post'))),
+                                'avatar_url' => $user->getAvatar(128, true),
+                                'title' => $language->get("user", "new_profile_post"),
+                                'url' => URL::getSelfURL() . 'profile/' . urlencode($profile_user->getDisplayname(true)) . '/#post-' . urlencode(DB::getInstance()->lastId())
```

Comment:
You need to use URL::Build after getSelfURL and remove / from it at the start, This does not support unfriendly urls

Use this `URL::getSelfURL() . ltrim(URL::build('/'profile/' . urlencode($profile_user->getDisplayname(true)) . '/#post-' . urlencode(DB::getInstance()->lastId())), '/'),`

Reply:
Thanks I've applied these changes.